cypress/integration/integration_spec.js

```diff
@@ -911,19 +911,9 @@ it('should submit full awards for all application', () => {
 
         cy.checkA11y();
 
-        cy.fixture('example.pdf', 'base64').then(fileContent => {
```

Comment:
Had to update this to use a new `attachFile` method in place of the old `upload` method. As you can this lets us go from manually loading a file and defining all the metadata for the upload to being able to simply pass the name of the fixture file, the module now handles the rest.